django/views/templates/technical_500.html

```diff
@@ -100,7 +100,7 @@
 <div id="summary">
   <h1>{% if exception_type %}{{ exception_type }}{% else %}Report{% endif %}
       {% if request %} at {{ request.path_info }}{% endif %}</h1>
-  <pre class="exception_value">{% if exception_value %}{{ exception_value|force_escape }}{% else %}No exception message supplied{% endif %}</pre>
+  <pre class="exception_value">{% if exception_value %}{{ exception_value|force_escape }}{% if exception_notes %}{{ exception_notes }}{% endif %}{% else %}No exception message supplied{% endif %}</pre>
```

Comment:
Given there's a `{{ exception_value|force_escape }}` on the line, do we need the same for `{{ exception_notes }}` here?

Reply:
Yes! Good catch :dart:

Reply:
OK, `force_escape` is unnecessary because `exception_notes` is always a string and it will always be escaped:
```python
c["exception_notes"] = "\n" + "\n".join(exc_notes)
```